crates/binjs_io/src/escaped_wtf8.rs

```diff
@@ -279,3 +303,222 @@ pub fn for_print(s: &SharedString) -> SharedString {
 
     SharedString::from_string(String::from_utf8(buf).expect("Escaped string should be valid UTF-8"))
 }
+
+/// Convert the given escaped_wtf8 string to JSON.  If the given string contains
```

Comment:
Shouldn't this be called `escape`?

Reply:
this returns string literal representation of the given string, including quotation,
that is compatible with JSON,
so I think `to_json` is more correct than `escape`.